bundles/org.eclipse.osgi.util/META-INF/MANIFEST.MF

```diff
@@ -2,23 +2,16 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %osgiUtil
 Bundle-SymbolicName: org.eclipse.osgi.util
-Bundle-Version: 3.6.100.qualifier
+Bundle-Version: 3.7.000.qualifier
```

Comment:
I think this should be `3.7.0.qualifier`.

Reply:
Done.